css-inline/src/lib.rs

```diff
@@ -74,6 +74,8 @@ pub struct InlineOptions<'a> {
     // Python wrapper for `CSSInliner` and `&str` in Rust & simple functions on the Python side
     /// Additional CSS to inline.
     pub extra_css: Option<Cow<'a, str>>,
+    /// Whether to break down styles into property tags
+    pub styles_as_props: bool,
```

Comment:
I think that it would be more precise to use "attributes" instead of "props" here and in other places :)

Reply:
I can certainly do that, I'll make the change